dvc/api.py

```diff
@@ -214,6 +215,268 @@ def read(path, repo=None, rev=None, remote=None, mode="r", encoding=None):
         return fd.read()
 
 
+def params_show(
+    *targets: str,
```

Comment:
Is it very relevant to pick by param files? Maybe the natural targets should be param names (return simple dict) or even stage names and this could be a secondary optional arg.

Reply:
> Is it very relevant to pick by param files?

In all `dvc {x} show` / `dvc {x} diff` commands, `targets` are files, i.e. https://dvc.org/doc/command-reference/metrics/show 

> Maybe the natural targets should be param names (return simple dict)

Not sure I follow what this would look like.

Reply:
I agree that stage names are likely most useful here, but we are also trying to mimic the CLI. It's a good idea @jorgeorpinel but there has already been a lot of related discussions and I would rather get something merged than try to optimize the order of args.

Reply:
Makes sense to follow the CLI.